### PR TITLE
Support resubmit action with CT

### PR DESCRIPTION
**Pull request by commandgjj**
In nicia extension, resubmit has two modes: with CT or without CT.
Resubmit with CT action is required if we want to implement stateful
firewall. This patch enables resubmit with CT flag support.

Signed-off-by: Jinjun Gao <gjinjun@gmail.com>

### CODE REVIEW

**Comment by commandgjj** [comment]
@hangyan @tnqn @srikartati @Jexf @wenyingd Can you please help review this patch? Thanks!

**Comment by commandgjj** [comment]
> @hangyan @tnqn @srikartati @Jexf @wenyingd Can you please help review this patch? Thanks!

@tnqn @wenyingd can you please help review this PR? Thanks!

**Comment by commandgjj** [comment]
> @hangyan @tnqn @srikartati @Jexf @wenyingd Can you please help review this patch? Thanks!

@wenyingd @tnqn can you help review this change? Thanks!

File: ofctrl/fgraphResubmit.go [hunk 1]
```diff
@@ -43,5 +48,22 @@ func NewResubmit(inPort *uint16, table *uint8) *Resubmit {
 	} else {
 		resubmit.nextTable = *table
 	}
+	resubmit.withCT = false
+	return resubmit
+}
+
+func NewResubmitWithCT(inPort *uint16, table *uint8) *Resubmit {
+	resubmit := new(Resubmit)
```
[the remark this answers is not in the document]
Comment (commandgjj) [reply ofctrl/fgraphResubmit.go:56]: Great! Will do it as you suggested. Thanks!

**Review by wenyingd — APPROVED** [review]
LGTM

**Pull request merged**